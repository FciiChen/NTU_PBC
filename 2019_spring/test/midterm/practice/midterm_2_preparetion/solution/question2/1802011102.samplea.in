John, Male, 35
Mary, Female, 45
Ashlee, Female, 12
Mia, Female, 75
Bob, Male, 12
BREAK
MARY, A12455
Bob, B55233
Mia, DEE123
BREAK