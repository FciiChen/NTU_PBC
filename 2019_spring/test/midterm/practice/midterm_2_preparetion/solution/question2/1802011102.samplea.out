John       Male       35         NULL      
Mary       Female     45         A12455    
Ashlee     Female     12         NULL      
Mia        Female     75         DEE123    
Bob        Male       12         B55233    
